src/KristofferStrube.Blazor.FileSystemAccess/Options/FileSystemAccessOptions.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Reflection;
+
+namespace KristofferStrube.Blazor.FileSystemAccess.Options;
+
+public class FileSystemAccessOptions
+{
+    public static readonly string DefaultNamespace = Assembly.GetExecutingAssembly().GetName().Name ?? "KristofferStrube.Blazor.FileSystemAccess";
+    public static readonly string DefaultPath = $"./_content/{DefaultNamespace}/{DefaultNamespace}.js";
+
+    public string ScriptPath { get; set; } = DefaultPath;
```

Comment:
We will need the base path for the future when creating objects from the `Blazor.FileAPI` and `Blazor.Streams` packages. By the base, I mean the `"./_content/"` path. So I suggest changing the option to also contain a BasePath property that defaults to `"./_content/"` from a static `DefaultBasePath` field and then changing the name of `DefaultPath` to `DefaultScriptPath` with just the value `"{DefaultNamespace}/{DefaultNamespace}.js"`.
This will of cause mean that we will need to concat `BasePath` and `ScriptPath` the places we use it.